### PR TITLE
fix: task def unique family

### PR DESCRIPTION
**Pull request by mohamed-cds**
Tested locally

### CODE REVIEW

File: runners/nuclei/entrypoint.sh [hunk 1]
```diff
@@ -1,5 +1,11 @@
 #!/bin/sh -l
 
+trap 'aws stepfunctions send-task-failure --task-token "${TASK_TOKEN_ENV_VARIABLE}" --task-output "Caught SIGTERM signal"' TERM
+trap 'aws stepfunctions send-task-failure --task-token "${TASK_TOKEN_ENV_VARIABLE}" --task-output "Caught SIGINT signal"' INT
+
+heartbeat_cmd="while sleep 60; do aws stepfunctions send-task-heartbeat --task-token ${TASK_TOKEN_ENV_VARIABLE}; done"
```
Comment (patheard) [inline runners/nuclei/entrypoint.sh:6]: Is there any chance of this infinitely looping?  I'm just wondering if there should be a break condition after some period of time that represents "something has gone weird".

File: runners/owasp-zap/entrypoint.sh [hunk 2]
```diff
@@ -3,6 +3,9 @@
 trap 'aws stepfunctions send-task-failure --task-token "${TASK_TOKEN_ENV_VARIABLE}" --task-output "Caught SIGTERM signal"' TERM
 trap 'aws stepfunctions send-task-failure --task-token "${TASK_TOKEN_ENV_VARIABLE}" --task-output "Caught SIGINT signal"' INT
 
+heartbeat_cmd="while sleep 60; do aws stepfunctions send-task-heartbeat --task-token ${TASK_TOKEN_ENV_VARIABLE}; done"
```
Comment (patheard) [inline runners/owasp-zap/entrypoint.sh:6]: Same question about infinite loops.

File: runners/nuclei/entrypoint.sh (hunk 1 above)
Comment (mohamed-cds) [reply runners/nuclei/entrypoint.sh:6]: Handled by the `TimeoutSeconds: 7200,` that's been added to the task with the understanding that the background task will not prevent the container from exiting. End condition will be added if that turns out to be false.

File: runners/owasp-zap/entrypoint.sh (hunk 2 above)
Comment (mohamed-cds) [reply runners/owasp-zap/entrypoint.sh:6]: Handled by the `TimeoutSeconds: 7200,` that's been added to the task with the understanding that the background task will not prevent the container from exiting. End condition will be added if that turns out to be false.

**Review by patheard — APPROVED** [review]

**Pull request merged**